cli/command/service/opts.go

```diff
@@ -510,7 +510,7 @@ func addServiceFlags(cmd *cobra.Command, opts *serviceOptions) {
 	flags.Var(&opts.restartPolicy.window, flagRestartWindow, "Window used to evaluate the restart policy")
 
 	flags.Uint64Var(&opts.update.parallelism, flagUpdateParallelism, 1, "Maximum number of tasks updated simultaneously (0 to update all at once)")
-	flags.DurationVar(&opts.update.delay, flagUpdateDelay, time.Duration(0), "Delay between updates")
+	flags.DurationVar(&opts.update.delay, flagUpdateDelay, time.Duration(0), "Delay between updates (Default none and Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h')")
```

Comment:
@dnephin address the case where the description is longer than 80-90 chars, and if your terminal has a small width the descriptions will wrap around and make everything hard to read.

Here's an example
<img width="639" alt="1" src="https://cloud.githubusercontent.com/assets/1525937/19939365/e3f05df0-a0e5-11e6-83ed-8ec1a07561ee.png">

It would be cool if the flag and description where treated as two separate columns, instead of a single column with a bunch of spaces between the flag and description.

|Flag|Description|
|----|-----|
|flag 1| Even if this description is long, it will wrap in its own column, and won't make `flag2` difficult to read|
|flag 2| Description of flag two|